playbooks/byo/openshift-master/restart.yml

```diff
@@ -0,0 +1,8 @@
+---
+- include: ../../common/openshift-master/restart.yml
+  vars:
```

Comment:
You should be able to use vars_files with playbooks/byo/openshift-cluster/cluster_hosts.yml instead of defining g_*_hosts